bash_scripts/bash_script/script.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+HOME=~/
```

Comment:
Please make your script more flexible and provide the ability to set variables(search folder, age parameter) without changing the script and user interaction. All parameters should be provided before running the script.

Reply:
Now script run should be with func name, examples:
1. ./script.sh dirs_older 1 ~/
2. ./script.sh files_older 1 ~/DevOps_TmS/
3. ./script.sh pc_monitoring
4. ./script.sh add_app_to_hosts
5. ./script.sh change_app_in_hosts
6. ./script.sh delete_file ~/